Shoot/Podfile

```diff
@@ -1,8 +1,8 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
 xcodeproj 'Shoot.xcodeproj'
-platform :ios, '7.0'
+platform :ios, '8.0'
 use_frameworks!
 
 pod 'AeroGearHttp', :git => 'http://github.com/aerogear/aerogear-ios-http'
-pod 'AeroGearOAuth2', :git => 'http://github.com/aerogear/aerogear-ios-oauth2'
+pod 'AeroGearOAuth2', :git => 'http://github.com/corinnekrych/aerogear-ios-oauth2-1', :branch => 'keycloak.1.9.1'
```

Comment:
should this be done on a branch, until the `aerogear-ios-oauth2` was released, instead of merging this to master ?